src/Controls/src/Core/IndicatorView/IndicatorStackLayout.cs

```diff
@@ -139,6 +141,7 @@ void AddExtraIndicatorItems()
 					HeightRequest = size,
 					CornerRadius = _indicatorView.IndicatorsShape == IndicatorShape.Circle ? (float)size / 2 : 0
 				};
+				indicator.BindingContext = items[i];
```

Comment:
Can we add the indicators as logical children to the indicatorview?

Reply:
do you mean call `AddLogicalChild(indicator);`?

Reply:
Apologies on the late response. Looking closer I realize that suggestion isn't correct. 

Out of curiosity, do you think this code could get slightly reworked to just use BindableLayout?  This class feels like it's kind of just re-implementing BindableLayout. 

I worry about other issues cropping up inside the IndicatorStackLayout that have already been considered by BindableLayout

Reply:
I am not familiar with BindableLayout. I will take a look if we can replace the layout.

Reply:
Yes it could @PureWeen